src/main/java/pages/LandingPage.java

```diff
@@ -0,0 +1,59 @@
+    package pages;
+
+    import org.openqa.selenium.By;
+    import org.openqa.selenium.WebDriver;
+    import org.openqa.selenium.interactions.Action;
+    import org.openqa.selenium.interactions.Actions;
+
+    public class LandingPage {
+
+
+        WebDriver driver;
+        private By addToCartsButton2 = By.cssSelector("a[title='Add to cart']");
+        private By continueShopping = By.cssSelector("span[title='Continue shopping'] span:nth-child(1)");
+        private By carts = By.cssSelector("a[title='View my shopping cart']");
+        private By carts2 = By.xpath("//b[normalize-space()='Cart']");
+        private By xRemove = By.xpath("//span[@class='remove_link']");
+        private By signInLink = By.cssSelector("a[title='Log in to your customer account']");
+        By womenLink = By.linkText("Women");
+
+
+
+        public LandingPage(WebDriver driver) {
+            this.driver = driver;
+        }
+        public void clickAddToCarts(){
+            driver.findElements(addToCartsButton2).get(1).click();
+            System.out.println("asdrrrr");
+        }
+
+        public void clickContinueShopping(){
+            driver.findElement(continueShopping).click();
+        }
+
+        public void removeFromCarts() throws InterruptedException {
+            Actions actions = new Actions(driver);
+            actions.moveToElement(driver.findElement(By.cssSelector("a[title='View my shopping cart']"))).build().perform();
```

Comment:
I noticed that you provide some path at the begining of the class, and some in the methods. Is there any reason for this? I think it is better that they are all together at the top of the class, so when you need to edit, they are all in one place.